src/test/java/org/threadly/litesockets/buffers/SimpleMergedByteBuffersTests.java

```diff
@@ -0,0 +1,277 @@
+package org.threadly.litesockets.buffers;
+
+import static org.junit.Assert.*;
+
+import java.nio.BufferUnderflowException;
+import java.nio.ByteBuffer;
+import java.util.Random;
+
+import org.junit.After;
+import org.junit.Test;
+import org.threadly.litesockets.buffers.MergedByteBuffers;
+import org.threadly.litesockets.buffers.ReuseableMergedByteBuffers;
+import org.threadly.litesockets.buffers.SimpleMergedByteBuffers;
+
+public class SimpleMergedByteBuffersTests {
```

Comment:
A lot of them already do, for the different SocketExecuters, I thought about it for this, but it does get a little complicated with how the MergedByteBuffers are constructed and added to differently.